packages/dds/sequence/src/sequence.ts

```diff
@@ -109,6 +110,10 @@ export interface ISharedSegmentSequenceEvents extends ISharedObjectEvents {
 	);
 }
 
+export interface SequenceOptions {
+	attribution: IMergeTreeAttributionOptions;
```

Comment:
i wonder if we want to take a firmer stance on what options are, specifically if they can be more than primitive values. probably doesn't need to be solved immediately.